jsonnet/components/grafana.libsonnet

```diff
@@ -35,60 +35,18 @@ function(params)
       ),
     },
 
-    trustedCaBundle: generateCertInjection.trustedCNOCaBundleCM(cfg.namespace, 'grafana-trusted-ca-bundle'),
-
-    // OpenShift route to access the Grafana UI.
-    route: {
-      apiVersion: 'v1',
-      kind: 'Route',
-      metadata: {
-        name: 'grafana',
-        namespace: cfg.namespace,
-      },
-      spec: {
-        to: {
-          kind: 'Service',
-          name: 'grafana',
-        },
-        port: {
-          targetPort: 'https',
-        },
-        tls: {
-          termination: 'Reencrypt',
-          insecureEdgeTerminationPolicy: 'Redirect',
-        },
-      },
+    config:: {
```

Comment:
It looks like we need to strip out most of the kube-prometheus jsonnet. Could we use github.com/brancz/kubernetes-grafana directly?